tsc "Can't find RowData"

Hi, while trying to transpile with tsc I get this error.

``` 
node_modules/embed-table/dist/structures/Row.d.ts:1:25 - error TS2307: Cannot find module 'index.js' or its corresponding type declarations.

1 import { RowData } from 'index.js';
                          ~~~~~~~~~~
```
![image](https://user-images.githubusercontent.com/6078221/231235085-fef19a13-6973-4c2b-823a-b82e6748cd64.png)


Using:
- embed-table@4.2.2
- node@18.15.0

--- a/src/structures/Row.ts
+++ b/src/structures/Row.ts
@@ -1,4 +1,4 @@
-import { RowData } from '../typings/index';
+import { RowData } from '../typings/index.js';
 
 export class Row {
 	/**

--- a/src/structures/Row.ts
+++ b/src/structures/Row.ts
@@ -1,4 +1,4 @@
-import { RowData } from 'index.js';
+import { RowData } from '../typings/index';
 
 export class Row {
 	/**

